deploy/crds/org_v1_che_cr.yaml

```diff
@@ -56,6 +56,27 @@ spec:
     workspaceNamespaceDefault: ''
     # defines if user is able to specify namespace different from the default
     allowUserDefinedWorkspaceNamespaces: false
+    # Sets the server and workspaces exposure type. Possible values are "multi-host", "single-host", "default-host".
+    # Defaults to "multi-host" which creates a separate ingress (or route on OpenShift) for every requied
+    # endpoint.
+    # "single-host" creates a single ingress/route and exposes workspaces on subpaths. Please read the docs
```

Comment:
this is true only if `singleHostWorkspaceExposureType=gateway` or defaults on OpenShift, right?

Reply:
I think it's still valid. `"single-host" creates a single ingress/route` is imho not true